test/test_route_utils.py

```diff
@@ -1,19 +0,0 @@
-import gradio as gr
```

Comment:
Fine to remove this as this was most testing implementation, but could we add some test to prevent this from regressing?

Reply:
Some end to end tests around `.select()` would be good -- we don't really test anywhere and we've had regressions pop up before